src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2958,20 +2958,20 @@
      *    |   |--VARIABLE_DEF
      *    |   |   |--MODIFIERS
      *    |   |   |--TYPE
-     *    |   |   |   `--LITERAL_INT (int)
-     *    |   |   `--IDENT (value)
+     *    |   |   |   +--LITERAL_INT (int)
```

Comment:
When I used the regex to make the example tree, it must've changed those.  I didn't notice. I will fix this now.  Should we make example tree like in CLI output?  That would make more sense than using `+`.

Reply:
The example you pointed me to used `+`, that's why I used that.  I think it would make more sense to make the tree look like it does in the CLI output, without line numbers.   Example:
```
`--LITERAL_INSTANCEOF -> instanceof 
      |--IDENT -> o 
      `--PATTERN_DEF -> PATTERN_DEF 
         `--VARIABLE_DEF -> VARIABLE_DEF 
               |--TYPE -> TYPE 
               |   `--IDENT -> String 
               `--IDENT -> s 
```

Reply:
Fixed.